### DISCUSSION TITLE
CFAST Subject Matter Expert (SME) or Consultant - Assistance with the Heat Release Rate (HRR) values

### DISCUSSION THREAD

**Discussion by aking36** [firemodels/cfast#2122] [General]
I am working on a project and I am in need of a CFAST Subject Matter Expert (SME) or Consultant. I would like assistance specifically in determining the Heat Release Rate (HRR) values that populate the time dependent graph on the Fires Tab within the software.

**Comment by mcgratta** [comment]
Heat Release Rate time histories can be found in journal papers, test reports, and reference books like the SFPE Handbook. You can also find HRR data in on-line databases like the one from [NIST](https://www.nist.gov/el/fcd) or [UL/FSRI](https://fsri.org/programs/materials-and-products-database).

**Reply by aking36** [reply]
Thanks for the response. I will keep looking to see what I can find. But I am still interested in finding a CFAST Subject Matter Expert (SME) or Consultant.
